packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx

```diff
@@ -0,0 +1,56 @@
+import FolderTreeExpandCollapse from './FolderTreeExpandCollapse?dev-site-example';
+
+# Expand/Collapse All
+
+### Description
+
+The following example demonstrates expand/collapse all functionality.
+Expand/collapse all functionality is accessible via the toolbar buttons.
+
+### Properties required
+
+Expanded/collapsed state is controlled by the consuming component.
+The following FolderTree props are required for expand/collapse all:
+* **onExpandAll**: This callback is called when interacting with the Expand All button.
```

Comment:
Same as above, probably should be links to the props table

Reply:
[4b462af](https://github.com/cerner/terra-framework/pull/1973/commits/4b462af48977e5fb036f4ed1a0a9666f2918c9de)